packages/recommendations-vdom/src/DefaultFallback.tsx

```diff
@@ -0,0 +1,7 @@
+/** @jsx createElement */
```

Comment:
it's a little confusing here because createElement isn't used as a prop. Should this maybe be inlined in where it's used, as it's not really a fallback, but nothing if it's not given

Reply:
I see it rather consistent with other default components. For now the default fallback is to not display anything but we're not sure if it'll stay like that.

I'd say let's keep it like that for now if you don't mind too much?